source/core/src/main/com/deco2800/game/areas/ForestGameArea.java

```diff
@@ -25,7 +25,7 @@
   private static final GridPoint2 PLAYER_SPAWN = new GridPoint2(10, 10);
   private static final float WALL_WIDTH = 0.1f;
   private static final String[] forestTextures = {
-    "images/box_boy_leaf.png",
+    "images/player_placeholders/BACK.png",
```

Comment:
Can we remove capitalization of the filenames of sprite assets?
I believe this MAY cause problems in Windows if some accidentally renames the file (ie. Back.png vs BACK.png).